{
    "containers": {
        "cna": {
            "affected": [
                {
                    "product": "Katello",
                    "vendor": "Katello",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "through 2013-07-12"
                        }
                    ]
                }
            ],
            "descriptions": [
                {
                    "lang": "en",
                    "value": "Katello has a Denial of Service vulnerability in API OAuth authentication"
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "description": "DoS in API OAuth authentication",
                            "lang": "en",
                            "type": "text"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "dateUpdated": "2019-12-10T14:32:37",
                "orgId": "53f830b8-0a3f-465b-8143-3b8a9948e749",
                "shortName": "redhat"
            },
            "references": [
                {
                    "tags": [
                        "x_refsource_MISC"
                    ],
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2013-4120"
                },
                {
                    "tags": [
                        "x_refsource_MISC"
                    ],
                    "url": "https://access.redhat.com/security/cve/cve-2013-4120"
                }
            ]
        },
        "adp": [
            {
                "providerMetadata": {
                    "orgId": "af854a3a-2127-422b-91ae-364da2661108",
                    "shortName": "CVE",
                    "dateUpdated": "2024-08-06T16:30:50.046Z"
                },
                "title": "CVE Program Container",
                "references": [
                    {
                        "tags": [
                            "x_refsource_MISC",
                            "x_transferred"
                        ],
                        "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2013-4120"
                    },
                    {
                        "tags": [
                            "x_refsource_MISC",
                            "x_transferred"
                        ],
                        "url": "https://access.redhat.com/security/cve/cve-2013-4120"
                    }
                ]
            }
        ]
    },
    "cveMetadata": {
        "assignerOrgId": "53f830b8-0a3f-465b-8143-3b8a9948e749",
        "assignerShortName": "redhat",
        "cveId": "CVE-2013-4120",
        "datePublished": "2019-12-10T14:32:37",
        "dateReserved": "2013-06-12T00:00:00",
        "dateUpdated": "2024-08-06T16:30:50.046Z",
        "state": "PUBLISHED"
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.1"
}